### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by sethi7ik**
A few minor bug fixes

### CODE REVIEW

File: Recipes/CGC/apps_copyFromMyProject.ipynb [hunk 1]
```diff
@@ -90,7 +90,7 @@
     "# [USER INPUT] Set project name; project (p_) and file (f_) indices here:\n",
     "source_project_name = 'Keep on Smiling'\n",
     "my_project_name = 'Life is Beautiful'\n",
-    "app_id = 'jack_digi/keep-on-smiling/TorrentMappingAlignmentProgramMap4/0'\n",
```
Comment (jackDiGi) [inline Recipes/CGC/apps_copyFromMyProject.ipynb]: So the bindings have fixed the issue of not defaulting to "latest" if not revision is specified?

File: Recipes/CGC/files_copyFromMyProject.ipynb [hunk 2]
```diff
@@ -107,7 +107,7 @@
     "\n",
     "# LIST all files in the source and target project\n",
     "my_files = api.files.query(limit = 100, project = my_project.id).all()\n",
-    "source_files = api.files.query(limit = 100, project = source_project.id).all()\n",
+    "source_files = list(api.files.query(limit = 100, project = source_project.id).all())\n",
```
Comment (jackDiGi) [inline Recipes/CGC/files_copyFromMyProject.ipynb]: So @SenadI specifically advised me not to use list() unless it was really needed.

The list-comprehension for **source_file** used to work well with the generator object, is it broken now or some other use I've missed?

File: Recipes/SBPLAT/apps_copyFromPublicApps.ipynb [hunk 3]
```diff
@@ -98,11 +98,289 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
```
Comment (jackDiGi) [inline Recipes/SBPLAT/apps_copyFromPublicApps.ipynb]: So you are choosing to show the cell outputs here as a helpful list for users. I think this is ok, but wanted to make sure this is a conscious choice.

File: Recipes/CGC/apps_copyFromMyProject.ipynb (hunk 1 above)
Comment (sethi7ik) [inline Recipes/CGC/apps_copyFromMyProject.ipynb]: No, I will change it to one with revision number.
The revision number within app ID causes an error as the app ID is not recognizable anymore. Revision number has to be given as a separate argument to the method now. I will change it.

File: Recipes/CGC/files_copyFromMyProject.ipynb (hunk 2 above)
Comment (sethi7ik) [inline Recipes/CGC/files_copyFromMyProject.ipynb]: Generators work as list when used only once in the code. Once you do this:
source_files = api.files.query(limit = 100, project = source_project.id).all()
source_file_names = [f.name for f in source_files]

source_files is empty (as it has been used once). As you do not have source_file initialized yet, the script had a bug. To get around that, I converted to list and did this:
source_file = [curr_file for curr_file in source_files if curr_file.name == source_file_names[f_index]][0]

To do that, you need to either recreate source_file_names with a new API call or make it a list earlier.  Hence, the suggested change. But [@SenadI](https://github.com/SenadI) can tell me what he recommends in this case.

File: Recipes/CGC/files_copyFromPublicReference.ipynb [hunk 4]
```diff
@@ -92,7 +92,7 @@
     "\n",
     "# LIST all files in the source and target project\n",
     "my_files = api.files.query(limit = 100, project = my_project.id).all()\n",
-    "source_files = api.files.query(limit = 100, project = source_project_id).all()\n",
+    "source_files = list(api.files.query(limit = 100, project = source_project_id).all())\n",
```
Comment (sethi7ik) [inline Recipes/CGC/files_copyFromPublicReference.ipynb]: same as above :)

File: Recipes/SBPLAT/apps_copyFromPublicApps.ipynb (hunk 3 above)
Comment (sethi7ik) [inline Recipes/SBPLAT/apps_copyFromPublicApps.ipynb]: I will remove this. Might have remained as a mistake.

File: Recipes/SBPLAT/volumes_writeToCloudStorage.ipynb [hunk 5]
```diff
@@ -224,7 +224,7 @@
    },
    "outputs": [],
    "source": [
-    "export_jobs = my_volume.get_exports(project=my_project)\n",
```
Comment (sethi7ik) [inline Recipes/SBPLAT/volumes_writeToCloudStorage.ipynb]: Thanks.

**Pull request merged**